cloud/client.go

```diff
@@ -141,13 +157,34 @@ func (c *Client) getRequestID() string {
 	return uuid.NewString()
 }
 
-func logstreamClient(ctx context.Context, defaultConn grpc.ClientConnInterface, overrideAddr string, dialOpts ...grpc.DialOption) (logstream.LogStreamClient, error) {
-	if overrideAddr == "" {
-		return logstream.NewLogStreamClient(defaultConn), nil
+var serviceConfig = `{
```

Comment:
this is so scary..... but https://pkg.go.dev/google.golang.org/grpc#WithDefaultServiceConfig confirms this is what's expected..... there be dragons :dragon: :ship: :see_no_evil:

Reply:
Yeah, I don't at all understand why there's only the string option for specifying the config. It should also be ok to express this in a struct.